### DISCUSSION TITLE
It Works!

### DISCUSSION THREAD

**Discussion by MikOfClassX** [JAndrassy/org.eclipse.team.cvs#2] [General]
HI, just few lines to say that this fix is perfectly working with Eclipse 2023-03.
Kudos and thanks for this.

Maybe you may submit the fix to the Eclipse devs.

Cheers!

**Comment by JAndrassy** [comment]
the Eclipse team stopped support for CVS. 
this is now the source repository for the plugin

**Comment by gaudryc** [comment]
Hi !

It works with Eclipse 2023-06, but I had to make the related feature directory (features/org.eclipse.cvs_1.4.2200.202304061448).

Great job. You save my team a lot of work.

Thank you.

**Comment by pmandi2** [comment]
Hi,

yes it works, thanks gaudryc for your post.

lg, Manfred

**Comment by matt-bamberger** [comment]
I got this installed successfully by exporting the feature to a local directory and using that as a local update site.

**Reply by JAndrassy** [reply]
a public update site exists now. see the README
`https://storage.googleapis.com/eclipse-cvs-plugin/`

**Reply by matt-bamberger** [reply]
OMG! I had added that update site, but it showed no items. I just realized it's because I had "group items by category" checked!
Thank you so much for making the necessary code changes to make this work and creating this repo!
